src/components/Message/OrderSuccess.js

```diff
@@ -0,0 +1,20 @@
+const OrderSuccess = ({close}) => {
```

Comment:
This one and the previous component looks identical except a message inside, so think about reusing component and pass the message as a prop